data_source/gateway_test.go

```diff
@@ -101,3 +103,151 @@ func TestAdaptStateUpdate(t *testing.T) {
 		}
 	}
 }
+
+func TestAdaptClass(t *testing.T) {
+	classJson, err := os.ReadFile("testdata/class.json")
```

Comment:
Please use go embed. There should be examples of it from other test files

Reply:
Also, please have a reference/link to the feeder gateway indicating where these transactions are coming from